tests/utils.go

```diff
@@ -1757,6 +1765,14 @@ func cleanNamespaces() {
 		for _, netDef := range nets.Items {
 			PanicOnError(virtCli.NetworkClient().K8sCniCncfIoV1().NetworkAttachmentDefinitions(namespace).Delete(context.Background(), netDef.GetName(), metav1.DeleteOptions{}))
 		}
+
+		// Remove all Istio Sidecars
+		err = removeAllGroupVersionResourceFromNamespace(schema.GroupVersionResource{Group: "networking.istio.io", Version: "v1beta1", Resource: "sidecars"}, namespace)
+		PanicOnError(err)
```

Comment:
You can inline this; the err is not being used anywhere.

Reply:
[Done](https://github.com/kubevirt/kubevirt/compare/ef7c453daccb75fb733dd6ef2c13636f0d5a51f7..5e074b4b40d1696b58a3c12d0802761156fcabe1)